lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua

```diff
@@ -643,21 +660,7 @@ if (SERVER) then
 			return true, #targets
 		end
 	end
-
-	local function initspawn(ply)
-		timer.Simple(10,function()
-			if (ply and ply:IsValid()) then
-				local bool, msg = EGP:SendDataStream( ply )
-				if (bool == true) then
-					ply:ChatPrint("[EGP] " .. tostring(msg) .. " EGP Screens found on the server. Sending objects now...")
-				end
-			end
-		end)
-	end
-
-	hook.Add("PlayerInitialSpawn","EGP_SpawnFunc",initspawn)
```

Comment:
@wrefgtzweve Probably caused by removing this

Reply:
Yeah that should probably only run for screens, it was entirely useless for huds. When i tried the regular streams managed fine and didnt cause me issues but i guess it slipped through